test/integration/app.js

```diff
@@ -57,21 +57,22 @@ describe('app', () => {
   describe('existing GP page', () => {
     it('should return a GP Page for a valid Org Code', (done) => {
       chai.request(app)
-        .get(`${constants.SITE_ROOT}/43213`)
+        .get(`${constants.SITE_ROOT}/44125`)
```

Comment:
With the logic for displaying the website in the view it would be good to have a test for when the website is available and when it is not available. The same goes for the other contact details. Maybe not as part of this PR but another one.